app/notifications/validators.py

```diff
@@ -69,11 +69,7 @@ def check_service_over_daily_message_limit(
     """
     # Enforce daily message limit only when configured
     # and the service is not using a test key.
-    if (
-        current_app.config['API_MESSAGE_LIMIT_ENABLED']
-        and current_app.config['REDIS_ENABLED']
-        and key_type != KEY_TYPE_TEST
-    ):
```

Comment:
Same here. I believe `API_MESSAGE_LIMIT_ENABLED` needs to stay. It's used in the lower envs (dev & staging) to enable daily message limits, which is not imposed in perf & prod.

Reply:
reverted